clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -9070,7 +9144,7 @@ static void ProcessDeclAttribute(Sema &S, Scope *scope, Decl *D,
     handleSYCLKernelAttr(S, D, AL);
     break;
   case ParsedAttr::AT_SYCLSimd:
-    handleSimpleAttribute<SYCLSimdAttr>(S, D, AL);
+    handleSYCLSimdAttr(S, D, AL);
     break;
```

Comment:
Once you make the `MutualExclusions` changes in Attr.td, this code can be reverted. However, as a drive-by (or follow up), Attr.td should be changed to specify `let SimpleHandler = 1;` for this attribute and then this code should be removed entirely.